### DISCUSSION TITLE
Shouldn´t this framework not implement some basic implementation of cookie compliance mechanism?

### DISCUSSION THREAD

**Discussion by sturlath** [abpframework/abp#11669] [Ideas]
There are [cookie stuff built in to .net](https://docs.microsoft.com/en-us/aspnet/core/security/gdpr?view=aspnetcore-6.0) and abp.io has all kinds of settings so I find it little odd that it doesn´t implement a basic rudimentary implementation of a compliance cookie settings mechanism.

What do you say?

**Comment by maliming** [comment]
This seems to only work in the EU, it's pretty simple to implement following Microsoft's documentation.
